app/components/TheMlm.vue

```diff
@@ -0,0 +1,59 @@
+<script setup lang="ts">
+import type { PropType } from 'vue'
+import type { Mlm } from '../types'
+import TheShareGrid from './TheShareGrid.vue'
+
+defineProps({
+  mlm: {
+    type: Object as PropType<Mlm>,
+    required: true,
+  },
+  feedbackUrl: {
+    type: String as PropType<string | null>,
+    default: null,
+  },
+})
+</script>
+
+<template>
+  <div class="flex flex-col h-full w-full text-center">
+    <div class="flex flex-col gap-8">
+      <h3 class="text-3xl dark:text-dark-white text-customBlack">
+        Is "{{ mlm.name }}" an MLM?
+      </h3>
+      <h3 class="text-3xl dark:text-dark-white text-customBlack">
+        Yes!
+      </h3>
+      <p class="h4 text-2xl">
+        Please check out
+        <nuxt-link to="https://en.wikipedia.org/wiki/Multi-level_marketing">
```

Comment:
Do these open in a new tab? I personally prefer when external links do not move you away from our website, as this makes us lose user retention.

Reply:
Sounds good! I updated it.